### PR TITLE
Fix #530 - Add null check while getting the last known location

### LINKED ISSUE #530: NPE when getting Location in Send Feedback flow
**Summary:** 

From Android Developer Console for staged rollout v2.0.12:

~~~
java.lang.NullPointerException: Attempt to invoke virtual method 'double android.location.Location.getLatitude()' on a null object reference
	at org.onebusaway.android.ui.HomeActivity.goToSendFeedBack(HomeActivity.java:1054)
	at org.onebusaway.android.ui.HomeActivity.goToNavDrawerItem(HomeActivity.java:411)
	at org.onebusaway.android.ui.HomeActivity.onNavigationDrawerItemSelected(HomeActivity.java:355)
	at org.onebusaway.android.ui.NavigationDrawerFragment.selectItem(NavigationDrawerFragment.java:263)
	at org.onebusaway.android.ui.NavigationDrawerFragment$3.onClick(NavigationDrawerFragment.java:430)
	at android.view.View.performClick(View.java:5197)
	at android.view.View$PerformClick.run(View.java:20926)
	at android.os.Handler.handleCallback(Handler.java:739)
	at android.os.Handler.dispatchMessage(Handler.java:95)
	at android.os.Looper.loop(Looper.java:145)
	at android.app.ActivityThread.main(ActivityThread.java:5942)
	at java.lang.reflect.Method.invoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1399)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1194)
~~~

User comment:

>Upgrade is terrible. No improvements, more confusing, can't turn off hints/help, crashes. Appears there was no testing by actual users.

**Steps to reproduce:** 

Unknown

**Expected behavior:** 

Not crash

**Observed behavior:** 

Crash

**Device and Android version:** 

~~~
Galaxy S4 (jfltevzw)
Manufacturer Samsung 
Android version Android 4.2 
RAM (MB) 2048 
Screen size 1080 × 1920 
Screen density (dpi) 480 
OpenGL ES version 2.0 
Native platform armeabi-v7a 
CPU make Qualcomm 
CPU model APQ8064
~~~

@cagryInside Could you please take a look?  Thanks!

### PR DESCRIPTION
**Pull request by cagryInside**
If the last known location is null then do not pass any location. MapView will ask to go to your service area by default:

device-2016-06-09-121109

### CODE REVIEW

File: onebusaway-android/src/main/java/org/onebusaway/android/report/ui/ReportActivity.java [hunk 1]
```diff
@@ -139,6 +139,19 @@ public static void start(Context context,
     }
 
     /**
+     * Starts the MapActivity with a particular stop focused with the center of
+     * the map at a particular point.
```
Comment (barbeau) [inline onebusaway-android/src/main/java/org/onebusaway/android/report/ui/ReportActivity.java]: @cagryInside I think this comment should be changed to reflect the lack of location?

Comment (cagryInside) [inline onebusaway-android/src/main/java/org/onebusaway/android/report/ui/ReportActivity.java]: Yes, actually the original method's comment is also wrong.

Comment (barbeau) [inline onebusaway-android/src/main/java/org/onebusaway/android/report/ui/ReportActivity.java]: Feel free to fix both :).

**Comment by cagryInside** [comment]
@barbeau comments should be fixed now.

**Comment by barbeau** [comment]
Thanks @cagryInside!

**Pull request merged**